roles/openshift-networking/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: ocp network operator
+  command: oc get networks.operator.openshift.io cluster -o json
+  register: ocp_network_operator
+
+- name: ocp dns config
+  command: oc get dns cluster -o json
+  register: ocp_dns
+
+- name: network attachment definitions
+  command: oc get network-attachment-definitions -A -o json
```

Comment:
`-A` is relatively new iirc, so would be best to replace with `--all-namespaces`